package.json

```diff
@@ -208,7 +208,7 @@
     "tinycolor2": "^1.4.1",
     "tlds": "^1.192.0",
     "unicode-substring": "^1.0.0",
-    "uws": "^10.148.0"
+    "uws": "10.148.0"
```

Comment:
Pegged to this version as there is a squabble going on. This package version is immutable